yoti_python_sdk/client.py

```diff
@@ -15,7 +15,7 @@
 from yoti_python_sdk.crypto import Crypto
 from yoti_python_sdk.endpoint import Endpoint
 from yoti_python_sdk.protobuf.v1 import protobuf
-from .config import SDK_IDENTIFIER
+from .config import *
```

Comment:
Is this is a good idea?  Is it intentional?

Reply:
Since I extracted all of the constants out, we would have to do `from .config import X_YOTI_SDK, SDK_IDENTIFIER, X_YOTI_SDK_VERSION, JSON_CONTENT_TYPE, X_YOTI_AUTH_KEY, X_YOTI_AUTH_DIGEST` instead. I thought it is neater this way. 

I know we should be wary of importing all, but given that these are just constants I don't think it's too problematic.